content/applications/inventory_and_mrp/manufacturing/shop_floor/shop_floor_tracking.rst

```diff
@@ -0,0 +1,119 @@
+========================
+Shop Floor time tracking
+========================
+
+.. |MO| replace:: :abbr:`MO (Manufacturing Order)`
+.. |MOs| replace:: :abbr:`MOs (Manufacturing Orders)`
+
+By signing in to the Odoo *Shop Floor* module as *operators*, employees are able to track the amount
+of time they spend working on each work order.
+
+Odoo tracks the time it takes to complete each work order, as well as the time each operator spends
+working on each work order.
+
+Operator sign in
+================
+
+To sign in to the *Shop Floor* module as an operator, sign in to the Odoo database, and open the
+:menuselection:`Shop Floor` module. The employee profile that is signed in to the database will
+automatically be signed in as an operator.
+
+All active operators are listed in the operator panel on the left side of the module. The panel can
+be opened or collapsed by clicking the :guilabel:`show/hide panel (white square with black column on
+left side)` button at the top-left corner of the module.
+
+.. image:: shop_floor_tracking/operator-panel.png
+   :align: center
+   :alt: The operator panel in the Shop Floor module, with the show/hide panel button above it.
+
+To sign in to *Shop Floor* as a different employee, click the :guilabel:`+ Add Operator` button at
+the bottom of the panel. Doing so opens the :guilabel:`Select Employee` pop-up window, which lists
+every employee that is able to sign in to the module.
+
+Click on a specific employee to sign in using their profile. If no PIN code is required to sign in
+as that employee, the profile will be signed in automatically.
+
+If a PIN code is required, a :guilabel:`Password?` pop-up window appears, showing a number pad on
+which the code can be entered. Enter the code using the number pad, and click :guilabel:`Confirm` to
+sign in to the *Shop Floor* module.
+
+.. image:: shop_floor_tracking/pin-code.png
+   :align: center
+   :alt: The "Password?" pop-up window, which is used to enter an operator PIN code.
+
+.. note::
+   A PIN code can be set for each employee, which must be entered each time they sign in to the
+   *Shop Floor* module, check in or out in the *Kiosk Mode* of the *Attendances* application, or sign
+   in as a cashier in the *Point of Sale* application.
+
+   To set an employee pin, navigate to the :menuselection:`Employees` app, and select a specific
```

Comment:
```suggestion
   To set an employee PIN, navigate to the :menuselection:`Employees` app, and select a specific
```
opting for capitalization here because "PIN" stands for 'Personal Identification Number'